<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NCom</class>
 <widget class="QWidget" name="NCom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ipAnyBox">
       <property name="text">
        <string>Any</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ipEdit">
       <property name="text">
        <string>195.0.0.13</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="portBox">
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>3000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disconnectButton">
       <property name="toolTip">
        <string>Disconnect</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/models/Icons/Disconnected-96.png</normaloff>:/models/Icons/Disconnected-96.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="toolTip">
        <string>Connect</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/models/Icons/Connected-96.png</normaloff>:/models/Icons/Connected-96.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="packetLabel">
     <property name="text">
      <string>Packets RX: 0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="posModeLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="posLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>288</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Map</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="mapPlotPointsBox">
        <property name="text">
         <string>Plot points</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mapDrawCarBox">
        <property name="text">
         <string>Draw car (ID 220)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
